src/archive.rs

```diff
@@ -226,6 +229,16 @@ where
             rt.spec_version,
             last_finalized_block
         );
+        if let Some(new_limit) = fdlimit::raise_fd_limit() {
```

Comment:
On windows `raise_fd_limit()` is always `None` which imo is worth mentioning in the docs somewhere, likely the README. Do you mind adding that in this PR?

Reply:
@dvdplm Done in https://github.com/paritytech/substrate-archive/pull/94/commits/757ffa95e913a03697a9157bd5db7e7870e99aec. Let me know if you want to put that into another section or something :(